src/CheckoutSdk/Sessions/Completion/CompletionInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+
+namespace Checkout.Sessions.Completion
+{
+    public abstract class CompletionInfo
+    {
+        [JsonProperty(PropertyName = "type")]
```

Comment:
not needed

Reply:
I had to add it because type is protected and without this task is not being serialized.